### PR TITLE
kth from end done

### PR DESCRIPTION
**Pull request by janiekyu**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: code401challenges/src/test/java/code401challenges/LinkedList/LinkedListTest.java [hunk 1]
```diff
@@ -52,4 +52,31 @@
...
+
+    @Test public void testkthFromEnd_kIsListLength() throws Exception {
+        assertEquals(1,multiNodeList.kthFromEnd(4));
+    }
+
+    @Test (expected = Exception.class)
+    public void testkthFromEnd_negativeK() throws Exception {
+        multiNodeList.kthFromEnd(-6);
+    }
+
```
Comment (Tw1stedRain) [inline code401challenges/src/test/java/code401challenges/LinkedList/LinkedListTest.java]: this test fails/throws an exception. please fix
